blacklight_advanced_search.gemspec

```diff
@@ -9,6 +9,7 @@ Gem::Specification.new do |s|
   s.email    = ["blacklight-development@googlegroups.com"]
   s.homepage = "http://projectblacklight.org/"
   s.summary  = "Blacklight Advanced Search plugin"
+  s.license  = "Apache 2.0"
```

Comment:
Should be `Apache-2.0` according to 
- http://guides.rubygems.org/specification-reference/#license= , referencing
- http://spdx.org/licenses/Apache-2.0.html